src/test/java/com/squareup/kotlinpoet/AbstractTypesTest.kt

```diff
@@ -211,16 +211,16 @@ abstract class AbstractTypesTest {
 
     override fun getKind() = declaredType.kind
 
-    override fun <R, P> accept(typeVisitor: TypeVisitor<R, P>, p: P)
-        = typeVisitor.visitError(this, p)
+    override fun <R, P> accept(typeVisitor: TypeVisitor<R, P>, p: P) =
+        typeVisitor.visitError(this, p)
 
-    override fun <A : Annotation> getAnnotationsByType(annotationType: Class<A>): Array<A>
-        = throw UnsupportedOperationException()
+    override fun <A : Annotation> getAnnotationsByType(annotationType: Class<A>): Array<A> =
+        throw UnsupportedOperationException()
 
-    override fun <A : Annotation> getAnnotation(annotationType: Class<A>): A
-        = throw UnsupportedOperationException()
+    override fun <A : Annotation> getAnnotation(annotationType: Class<A>): A =
+        throw UnsupportedOperationException()
 
-    override fun getAnnotationMirrors()
-        = throw UnsupportedOperationException()
+    override fun getAnnotationMirrors() =
```

Comment:
nit: this can fit on a single line

Reply:
5fca64f772bad7052c1836d3fde8e485210d4395